test/bin/run-integration-tests.sh

```diff
@@ -48,6 +48,7 @@ PYTHONPATH="$( pwd )" \
     python test/bin/inject_api_key.py admin@user.com \
     "XZpXI40Uk85eozjQkU1zHJ6yZHpix+j0mo1TMeGZ4dPzIqVPVGPmyfeK"
 
+PYTHONPATH="$( pwd )" \
```

Comment:
@nagem Sweet. Once that's in I can hook up the parameter passing from `run-tests-ubuntu.sh` unless you want to tackle that as well with your change.